shared/js/ui/models/grade-scorecard.es6.js

```diff
@@ -0,0 +1,75 @@
+const Parent = window.DDG.base.Model
+
+function GradeScorecard (attrs) {
+  Parent.call(this, attrs)
+
+  this.reasons = this._getReasons()
+}
+
+GradeScorecard.prototype = window.$.extend({},
+  Parent.prototype,
+  {
+    modelName: 'gradeScorecard',
+
+    _getReasons: function () {
+      const site = this.site
+      let reasons = []
+
+      // grab all the data from the site to create
+      // a list of reasons behind the grade
+
+      // encryption status
+      let httpsStatusText = site.httpsStatusText
```

Comment:
The way I've been working with `let` vs. `const` is: if its a variable that gets updated after being declared then its a `let`. Otherwise, it should be a `const`. In this case, `httpsStatusText` appears to be a const. Please review the rest of the `let`s in this function likewise.

Reply:
Thank you for pointing this out - I'm still internalizing how to use `let` vs `const`!